Added BasisCode Compliance Software to other.yml

baileybasiscode opened the pull request:
Added BasisCode Compliance Software because it has 2FA and contains private and secure information

baileybasiscode committed:
Added BasisCode Compliance Software to other.yml

Added BasisCode Compliance Software because it has 2FA and contains private and secure information

Jawshy reviewed (changes requested):
Thanks for submitting a pull request, @baileybasiscode. Could you please add an image for the site? The image will need to be 32 by 32 pixels, PNG format, and under 2,500 bytes in size. 

[Travis CI Build](https://travis-ci.org/2factorauth/twofactorauth/jobs/182165605)
```
BasisCode Compliance image not found.
```

Jawshy committed:
other: BasisCode Compliance: Clean

Jawshy commented on _data/other.yml:
```diff
@@ -72,6 +72,14 @@ websites:
 
+    - name: BasisCode Compliance
+      url: http://basiscode.com
+      img: http://basiscode.com/wp-content/uploads/2016/10/BasisCode-Compliance.png
```
See [FAQ_New_TFA_SITE](https://github.com/2factorauth/twofactorauth/wiki/FAQ_New_TFA_SITE) for how to correctly add an image.

Jawshy commented on _data/other.yml:
```diff
@@ -72,6 +72,14 @@ websites:
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      phone: Yes
```
Is there any public documentation for how two-factor authentication is implemented on the site?

baileybasiscode committed:
Added a reference to the new basiscode.png

The new basiscode.png is located in the img/other/basiscode.png

On online version can be found here: http://basiscode.com/wp-content/uploads/2016/12/compLogo32x32.png

If this still doesn't work, feel free to let me know. 

Thanks in advance for giving us the opportunity to put our site here.

Joe

baileybasiscode committed:
Added basiscode.png to img/other

baileybasiscode committed:
Delete wrong sized basiscode.png

baileybasiscode committed:
uploaded new basiscode.png with the correct size

Carl,

Thanks.. .Uploaded the new graphic with the correct size.

Jawshy reviewed (dismissed):
> Is there any public documentation for how two-factor authentication is implemented on the site? 

See [FAQ_New_TFA_SITE](https://github.com/2factorauth/twofactorauth/wiki/FAQ_New_TFA_SITE).

Carlgo11 reviewed (dismissed):
Needs a new logo as requested in [266583020](https://github.com/2factorauth/twofactorauth/pull/2223#issuecomment-266583020) and [266587168](https://github.com/2factorauth/twofactorauth/pull/2223#issuecomment-266587168).

baileybasiscode committed:
Uploaded a new image file for BasisCode

Thanks again for the help.

baileybasiscode committed:
Basiscode.com Added url to 2FA Doc

Again, thanks. This latest update includes our marketing on the 2FA offered in the software. As always, if you have any questions, suggestions, let me know.

Thanks,
Joe

Jawshy reviewed (changes requested):
> Ok, I just submitted 2 pull requests on changeset 3 which includes a new graphic and a url to the 2FA documentation. Feel free to let me know if there are any issues. Thanks again. Joe 

[Travis CI is having an issue](https://travis-ci.org/2factorauth/twofactorauth/jobs/185297262) with the [image that was used](https://github.com/baileybasiscode/twofactorauth/blob/9d5fab73bdb25294a6c846c20543704c517022c8/img/other/basiscode.png). Could you please replace it with the below image, @baileybasiscode? 

![basiscode-resaved](https://cloud.githubusercontent.com/assets/10876578/21336166/4cd1b456-c6af-11e6-8d95-604411f4bb5f.png)

baileybasiscode committed:
Travis CI Recommended graphic uploaded

Thanks again! Included in this push is the recommended graphic from Travis CI.
Joe

Jawshy committed:
img: other: basiscode: Updated image

Enlarged the site's favicon from 16² px to 32² px. Should have done this
from the start.
